docs/integrations/destinations/databricks.md

```diff
@@ -5,335 +5,74 @@
-[documentation](https://docs.databricks.com/data/tables.html#managed-and-unmanaged-tables) for
-details. In summary, you have full control of the location of the data underlying an unmanaged
-table. In S3, the full path of each data stream is:
-
-```text
-s3://<bucket-name>/<bucket-path>/<database-schema>/<stream-name>
-```
-
-For example:
-
-```text
-s3://testing_bucket/data_output_path/public/users
-     ↑              ↑                ↑      ↑
-     |              |                |      stream name
-     |              |                database schema
-     |              bucket path
-     bucket name
-```
-
-In Azure, the full path of each data stream is:
-
-```text
-abfss://<container-name>@<account-name>.dfs.core.windows.net/<database-schema>/<stream-name>
-```
-
-Please keep these data directories on S3/Azure. Otherwise, the corresponding tables will have no
-data in Databricks.
-
 ## Output Schema
 
-Each table will have the following columns:
+Each table will have the following columns, in addition to your whatever columns were in your data:
 
-| Column                |   Type    | Notes                                            |
-| :-------------------- | :-------: | :----------------------------------------------- |
-| `_airbyte_ab_id`      |  string   | UUID.                                            |
-| `_airbyte_emitted_at` | timestamp | Data emission timestamp.                         |
-| `_airbyte_data`       |   JSON    | The data from your source will be in this column |
+| Column                   |   Type    | Notes                                                                  |
+| :----------------------- | :-------: | :--------------------------------------------------------------------- |
+| `_airbyte_raw_id`        | string    | A random UUID.                                                         |
+| `_airbyte_extracted_at`  | timestamp | Timestamp when the source read the record.                             |
+| `_airbyte_loaded_at`     | timestamp | Timestamp when the record was written to the destination               |
+| `_airbyte_generation_id` | bigint    | See the [refreshes](../../operator-guides/refreshes.md) documentation. |
 
-Under the hood, an Airbyte data stream in Json schema is first converted to an Avro schema, then the
-Json object is converted to an Avro record, and finally the Avro record is outputted to the Parquet
-format. Because the data stream can come from any data source, the Json to Avro conversion process
-has arbitrary rules and limitations. Learn more about how source data is converted to Avro and the
-current limitations [here](https://docs.airbyte.com/understanding-airbyte/json-avro-conversion).
+Airbyte will also produce "raw tables" (by default in the `airbyte_internal` schema). We do not recommend directly interacting
+with the raw tables, and their format is subject to change without notice.
 
 ## Related tutorial
 
```

Comment:
Content of the link in related tutorial is stale, that doc is probably in Webflow or somthing ? https://airbyte.com/tutorials/load-data-into-delta-lake-on-databricks-lakehouse

Reply:
ah, didn't catch that. Yeah I"ll remove this section for now